lib/stripe/api.ex

```diff
@@ -19,6 +19,15 @@ defmodule Stripe.API do
   @api_version "2018-08-23"
   @http_module Application.get_env(:stripity_stripe, :http_module) || :hackney
 
+  @doc """
+  In config.exs your implicit or expicit configuration is:
+    config :stripity_stripe,
+      json_library: Jason # defaults to Poison but can be configured to Jason
+  """
+  def json_library() do
+    Application.get_env(:stripity_stripe, :json_library, Poison)
```

Comment:
We could remove the `Poison` dependency and use something like `Code.ensure_loaded`, thus leaving it on the end user to have a json library installed.  What do you think?

Reply:
Although this is a breaking change so perhaps needs to happen in another release.